code/style.css

```diff
@@ -0,0 +1,314 @@
+
+.marginTop {
+    margin-top: 150px;
+}
+
+/*  PROGRESS-BAR  */ 
+.progress-bar-container-outer {
+    position: fixed;
+    left: 0;
+    top: 0;
+    width: 100vw;
+    height: 95px;
+    background: #000;
+    border-bottom: 2px solid #01DC3E;
+    transition: visibility 0s, opacity 0.5s linear;
+}
+
+.progress-bar-container-inner {
+    width: 70vw;
+    margin: auto;
+    padding: 20px;
+    font-size: 12px;
+}
+
+.progress-bar-container-inner p {
+    margin: 5px 0;
+}
+
+.progress-bar-container-inner-content {
+    display: flex;
+    justify-content: space-between;
+}
+
+.progress-bar {
+    width: 100%;
+    display: flex;
+}
+
+.progress-bar div {
+    height: 1vh;
+    width: 50%;
+    background: #333;
+}
+
+/*  CHAT-BOXES  */ 
+.chat-box {
+    display: flex;
+    align-items: center;
+    margin: 10px 0;
+    transition: opacity 0.5s linear;
+}
+
+.pizzeria p {
+    color: #fff;
+    background: #01DC3E;
+    width: 30vw;
+    padding: 20px;
+    border-radius: 10px;
+    margin-left: 20px;
+}
+
+.user {
+    align-self: flex-end;
+}
+
+.user-inner {
+    padding: 15px 20px 15px 20px;
+    border-radius: 10px;
+    color: #fff;
+    border: 2px solid #999;
+    margin-right: 20px;
+}
+
+/*  PORTRAITS  */ 
+.chat-portrait {
+    width: 80px;
+    height: 80px;
+    border: 2px solid #01DC3E;
+    border-radius: 50%;
+    background-image: url(./images/pizza-slice2.png);
+    background-size: 30px;
+    background-position: center;
+    background-repeat: no-repeat;
+}
+
+.chat-portrait-user {
+    width: 80px;
+    height: 80px;
+    border: 2px solid #999;
+    border-radius: 50%;
+    background-image: url(./images/user.png);
+    background-size: 30px;
+    background-position: center;
+    background-repeat: no-repeat;
+}
+
+.chat-form {
+    display: flex;
+    flex-direction: row;
+    margin-top: 15px;
+}
+
+/*  FORM  */ 
+input { /* The different pizza-inputs to choose among.*/
+    font-family: 'Roboto Mono', monospace;
+    color: #fff;
+    background: #01DC3E;
+    padding: 10px;
+    cursor: pointer;
+    border: none;
+    box-shadow: 1px 1px #004D16;
+    text-shadow: 1px 1px #004D16;
+}
+
+#quantityUser { /* The box where the user set number of pizzas */
+    background: #fff;
+    color: #000;
+    text-shadow: none;
+}
+
+button { /* Button to confirm quantity */
+    font-family: 'Roboto Mono', monospace;
+    color: #fff;
+    background: #01DC3E;
+    cursor: pointer;
+    border: none;
+    box-shadow: 1px 1px #004D16;
+    text-shadow: 1px 1px #004D16;
+    padding: 10px;
+    margin: 5px;
+}
+
+button:hover, input:hover { /*When hovering button*/
+    background: #00C337;
+}
+
+/* COUNTDOWN TIMER */
+.timerBox {
+    width: auto;
+    border: 1px solid #fff;
+    margin-top: 40px;
+    padding: 20px;
+    align-self: center;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+}
+
+.time {
+    font-size: 50px;
+    align-self: center;
+}
+
+/*  ORDER SUMMARY  */ 
+.orderSummaryBox {
```

Comment:
classes should not be written in `camelCase` convention but in `kebab-case` one

Reply:
@EmOlsz 
We see that we have had some different methods that we will change, but  to calrify what we have done wrong... 
as I understood it: In HTML we write the class names in `kebab-case` and id names in` camelCase`.
For example: <div id="summaryBox" class="summary-box"
Is that right or could you clarify a bit more? :)

Reply:
As Van mentioned yesterday during lecture, naming id's by `camelCase` and `kebab-case` is almost 50/50.
Classes should be named by `kebab-case` in 100% of occurrences
Id's should be named by `kebab-case` right now, but be aware that you might find yourself looking at someone else code where id's will be done by `camelCase` and it's also OK